remarkable_update_image/image.py

```
@@ -82,86 +98,99 @@ def __init__(self, update_file, cache_size=500, cache_ttl=60):
 
-            size = struct.unpack(">Q", f.read(8))[0]
+            size = cast(int, struct.unpack(">Q", f.read(8))[0])
             data = f.read(size)
-            self._manifest = DeltaArchiveManifest.FromString(data)
-            self._offset = f.tell()
+            self._manifest: DeltaArchiveManifest = DeltaArchiveManifest.FromString(data)  # pyright: ignore[reportUnknownMemberType]
+            self._offset: int = f.tell()
 
-        for blob, offset, length, f in self._blobs:
+        self._size: int = 0
+        for _, _, length, f in self._blobs:  # pyright: ignore[reportUnknownMemberType, reportUnknownVariableType]
```

Ruff (0.15.12)

[warning] Loop control variable `f` not used within loop body

(B007)